client/src/components/dashboard/__snapshots__/DashboardHeader.test.js.snap

```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `
+<div
+  className="sc-bdVaJa ePNJYe"
+>
+  <em
+    className="sc-bwzfXH iguHiQ"
+  >
+    Survey Details
+  </em>
+  <em
+    className="sc-bwzfXH iguHiQ"
+  >
+    Response Rate
+  </em>
+</div>
+`;
```

Comment:
Just a small comment - but this HTML is looking pretty ugly. 

Is it possible to fix up to make look like:
```
<div className="sc-bdVaJa ePNJYe">
  <em className="sc-bwzfXH iguHiQ">
    Survey Details
  </em>
```

Or is this some react rendering thing?

Reply:
This code is generated by Jest as a snapshot, and is used only for testing against the rendering of markup by components.